### PR TITLE
Ambiguous Grammatical Update github-flow.md

### PR DESCRIPTION
**Pull request by KaladinB4**
'Address review comments' is a bit difficult to understand -- more unambiguous

### Why:

Closes: 

### Check off the following:

- [ ] I have reviewed my changes in staging, available via the **View deployment** link in this PR's timeline (this link will be available after opening the PR).

  - For content changes, you will also see an automatically generated comment with links directly to pages you've modified. The comment won't appear if your PR only edits files in the `data` directory.
- [ ] For content changes, I have completed the [self-review checklist](https://docs.github.com/en/contributing/collaborating-on-github-docs/self-review-checklist).

### CODE REVIEW

**Comment by nguyenalex836** [comment]
@KaladinB4 Thanks so much for opening a PR! ✨ Can you elaborate on how "address review comments" is more difficult to understand than "address reviewer comments"? 💛

File: content/get-started/using-github/github-flow.md [hunk 1]
```diff
@@ -71,7 +71,7 @@ Your repository may be configured to automatically request a review from specifi
 
 If your repository has checks configured to run on pull requests, you will see any checks that failed on your pull request. This helps you catch errors before merging your branch. For more information, see "[AUTOTITLE](/pull-requests/collaborating-with-pull-requests/collaborating-on-repositories-with-code-quality-features/about-status-checks)."
 
-### Address review comments
+### Address reviewer comments
```
Comment (Alfamoda) [inline content/get-started/using-github/github-flow.md:74]: Very fashionable products

**Review by Alfamoda — COMMENTED** [review]
Very fashionable products

**Comment by Mook90** [comment]
Good effort

File: content/get-started/using-github/github-flow.md (hunk 1 above)
Comment (Mook90) [reply content/get-started/using-github/github-flow.md:74]: Amazing

**Comment by Qandosindiy** [comment]
Sir really I don't know which one it's true

Android için Outlook<https://aka.ms/AAb9ysg> edinin
________________________________

File: content/get-started/using-github/github-flow.md (hunk 1 above)
Comment (Jennykehe) [reply content/get-started/using-github/github-flow.md:74]: content/get-started/using-github/github-flow.md

**Comment by Mook90** [comment]
________________________________ From: Mook90 ***@***.***> Sent: Saturday, March 16, 2024 6:25:17 AM To: github/docs ***@***.***> Cc: Subscribed ***@***.***> Subject: Re: [github/docs] Ambiguous Grammatical Update github-flow.md (PR #32108) @Mook90 commented on this pull request.

________________________________ In content/get-started/using-github/github-flow.md<#32108 (comment)>:

@@ -71,7 +71,7 @@ Your repository may be configured to automatically request a review from specifi

If your repository has checks configured to run on pull requests, you will see any checks that failed on your pull request. This helps you catch errors before merging your branch. For more information, see "[AUTOTITLE](/pull-requests/collaborating-with-pull-requests/collaborating-on-repositories-with-code-quality-features/about-status-checks)." -### Address review comments +### Address reviewer comments Amazing

File: content/get-started/using-github/github-flow.md (hunk 1 above)
Comment (Mook90) [reply content/get-started/using-github/github-flow.md:74]: https://accounts.google.com/v3/signin/confirmidentifier?authuser=1&btmpl=mobile&continue=https%3A%2F%2Fmail.google.com%2Fmail%2Fmu%2Fmp%2F431%2F%3Flogin%3D1&followup=https%3A%2F%2Fmail.google.com%2Fmail%2Fmu%2Fmp%2F431%2F%3Flogin%3D1&ifkv=ATuJsjyAc3ea6eaA1SVWYOqJVFCP-Ejknzu8KMMkRkEclomgVPG9pRY5nglFv5euYBXGi50clbCH&ltmpl=mobile&osid=1&passive=1209600&service=mail&flowName=GlifWebSignIn&flowEntry=ServiceLogin&dsh=S-368751415%3A1710562606037038&theme=mn&ddm=0#tl/priority/%5Eio_im

**Comment by Mook90** [comment]
Need help i get hacked from my github acoount

**Comment by Mook90** [comment]
https://github.com/github/docs/pull/32108

**Comment by Mook90** [comment]
Heeeey

**Comment by V3ntus** [comment]
> Need help i get hacked from my github acoount

I'm just baffled how people like you exist on the internet

**Comment by Mook90** [comment]
Becouse you r here

**Comment by Mook90** [comment]
Need more review

**Pull request closed without merging**